python/python/glide/async_commands/server_modules/ft_constants.py

```diff
@@ -0,0 +1,32 @@
+# Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
```

Comment:
please move this to ft_options, top lever dir should only contain the commands / modules

Reply:
Updated.

All changes suggested in all comments in this review are part of this commit: 
https://github.com/valkey-io/valkey-glide/pull/2437/commits/02638137f7577b3643ef6d78a7c9b266d96b3d7f